scripts/common/object_store.sh

```diff
@@ -241,22 +241,17 @@ function migrate_rgw_to_minio_checks() {
     fi
 
     log "Wating for Rook Ceph Object Store health ..."
-    if ! spinner_until 300 rook_rgw_is_healthy ; then
+    if ! spinner_until 300 rook_rgw_check_if_is_healthy ; then
         logFail "Failed to detect healthy Rook Ceph Object Store"
         bail "Cannot upgrade from Rook Ceph Object Store to Minio. Rook Ceph is unhealthy."
     fi
 
     logSuccess "Rook Ceph Object Store to Minio migration checks completed successfully."
 }
 
-function rook_rgw_is_healthy() {
-    export OBJECT_STORE_CLUSTER_IP
```

Comment:
IIUC the problem was that we were overwriting the `OBJECT_STORE_CLUSTER_IP`, is that correct ?